rotkehlchen/api/rest.py

```diff
@@ -1555,6 +1557,94 @@ def process_history(
         result_dict = _wrap_in_result(result=result, message=msg)
         return api_response(result_dict, status_code=status_code)
 
+    def _get_history_debug(
+        self,
+        from_timestamp: Timestamp,
+        to_timestamp: Timestamp,
+    ) -> Dict[str, Any]:
+        """This method creates all history events for a timestamp range.
+        It also gets the user settings & ignored action identifiers all to be exported
+        for PnL debugging.
+        """
+        error_or_empty, events = self.rotkehlchen.events_historian.get_history(
+            start_ts=from_timestamp,
+            end_ts=to_timestamp,
+            has_premium=self.rotkehlchen.premium is not None,
+        )
+        if error_or_empty != '':
+            return wrap_in_fail_result(error_or_empty, status_code=HTTPStatus.CONFLICT)
+
+        settings = self.rotkehlchen.get_settings()
+        ignored_ids = self.rotkehlchen.data.db.get_ignored_action_ids(None)
+        debug_info = {
+            'events': [entry.serialize() for entry in events],
+            'settings': process_result(settings),
+            'ignored_events_ids': {str(k): v for k, v in ignored_ids.items()},
+        }
+        return {'result': debug_info, 'message': error_or_empty}
+
+    def get_history_debug(
+        self,
+        from_timestamp: Timestamp,
+        to_timestamp: Timestamp,
+        async_query: bool,
+    ) -> Response:
+        if async_query is True:
+            return self._query_async(
+                command=self._get_history_debug,
+                from_timestamp=from_timestamp,
+                to_timestamp=to_timestamp,
+            )
+
+        response = self._get_history_debug(
+            from_timestamp=from_timestamp,
+            to_timestamp=to_timestamp,
+        )
+        status_code = _get_status_code_from_async_response(response)
+        result_dict = _wrap_in_result(result=response['result'], message=response['message'])
+        return api_response(result_dict, status_code=status_code)
+
+    def _import_history_debug(self, file: Union[FileStorage, Path]) -> Dict[str, Any]:
```

Comment:
As discussed while solving the csv importing tmp directory problem, `file` is not a good variable name since in the past it was a reserved python keyword